frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -60,7 +60,7 @@ const UpdatesAvailableMessage: React.SFC<CVStatusMessageProps> = ({cv}) => <Reac
     <i className="fa fa-arrow-circle-o-up update-pending" aria-hidden={true} /> Update available
   </div>
   <div>
-    <Button bsStyle="primary" onClick={() => clusterUpdateModal(cv)}>
+    <Button bsStyle="primary" onClick={() => clusterUpdateModal({cv})}>
```

Comment:
Line 53 is incorrect, too?

Reply:
That one looks right. `cancelUpdate` takes `cv` as an arg, not through props.